### PR TITLE
`ethereum-light-client`: add wasm functions to use in eth lc typescript sdk

### PR DESCRIPTION
**Pull request by yuvalelmo**

### CODE REVIEW

File: sdk/eth-light-client-wasm/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+use serde::{Deserialize, Serialize};
+use std::str::FromStr;
+use wasm_bindgen::prelude::wasm_bindgen;
+use wasm_bindgen::JsValue;
+
+#[derive(Serialize, Deserialize)]
+/// Error type for better JS handling and generalization
+/// of Rust / WASM -> JS error conversion.
+pub struct JsErr {
+    // type_: String,
```
Comment (yuvalelmo) [inline sdk/eth-light-client-wasm/src/lib.rs:14]:
```suggestion
```

File: sdk/eth-light-client-wasm/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+/// * `network` - A `JsValue` representing the network as a string.
+/// * `bootstrap` - A `JsValue` representing the bootstrap configuration.
+///
+/// # Returns
+///
+/// A `Result` containing a `JsValue` with the serialized Ethereum state value or a `JsErr` on failure.
+#[wasm_bindgen]
+pub fn get_initial_state_bcs(
+    checkpoint: JsValue,
+    rpc: JsValue,
```
[the remark this answers is not in the document]
Comment (yuvalelmo) [reply sdk/eth-light-client-wasm/src/lib.rs:68]: This is used to initialize the `ConsensusStateManager`

[the remark this answers is not in the document]
Comment (ItayLevyOfficial) [reply sdk/eth-light-client-wasm/src/lib.rs:68]: Ye, but why can't you use some constant, dummy value?

File: sdk/eth-light-client-wasm/LICENSE
Comment (yaelAbergel) [inline sdk/eth-light-client-wasm/LICENSE:1]: I'm not familiar with the conventions for LICENSE files, but the lines here are extremely long, which makes it harder to read.

File: sdk/eth-light-client-wasm/README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+## Build locally
+
+To build the binary, you need to have Rust installed and then the `wasm-pack`. The installation script [can be found here](https://rustwasm.github.io/wasm-pack/).
```
Comment (yaelAbergel) [inline sdk/eth-light-client-wasm/README.md:3]:
```suggestion
To build the binary, you need to have Rust installed and then the `wasm-pack`.   
The installation script [can be found here](https://rustwasm.github.io/wasm-pack/).
```
Could you add two spaces at the end of the line and break it to improve readability for people with screens slightly smaller than yours? :)

**Review by yaelAbergel — APPROVED** [review]

File: sdk/eth-light-client-wasm/LICENSE
Comment (yuvalelmo) [reply sdk/eth-light-client-wasm/LICENSE:1]: Actually I am not familiar with that either. I took this license file from the `signature-mpc-wasm` package

**Review by ycscaly — APPROVED** [review]

**Pull request merged**